src/Runners/PHPUnit/Options.php

```diff
@@ -219,6 +219,8 @@ final class Options
     private $orderBy;
     /** @var int */
     private $randomOrderSeed;
+    /** @var string|null */
```

Comment:
```suggestion
    /** @var int */
```

Reply:
Will we enforce always sending the repeat option to PHPUnit? If so, we need to set default to '1', no?

Reply:
No: the option in passed only within the `if {}` with the  `$filters` variable.
The `Options::$repeat` attribute is here only as a reference of the CLI arguments

Reply:
Oh, okay.